tutorials/platform/ios/ios_plugin.rst

```diff
@@ -80,18 +84,21 @@ To build an iOS plugin:
 
             files=["data.json"]
 
+            linker_flags=["-ObjC"]
+
             [plist]
             PlistKey="Some Info.plist key you might need"
 
         The ``config`` section and fields are required and defined as follow:
 
             -   **name**: name of the plugin
 
-            -   **binary**: this should be the filepath of the plugin ``a`` file.
+            -   **binary**: this should be the filepath of the plugin library (``a`` or ``xcframework``) file.
 
-                -   The filepath can be relative (e.g.: ``MyPlugin.a``) in which case it's relative to the directory where ``gdip`` file is located.
-                -   The filepath can be absolute: ``res://some_path/MyPlugin.aar``.
-                -   In case you need multitarget library usage, filename should be ``MyPlugin.a`` and ``a`` files should be name as ``MyPlugin.release.a`` and ``MyPlugin.debug.a``.
+                -   The filepath can be relative (e.g.: ``MyPlugin.a``, ``MyPlugin.xcframework``) in which case it's relative to the directory where ``gdip`` file is located.
+                -   The filepath can be absolute: ``res://some_path/MyPlugin.a`` or ``res://some_path/MyPlugin.xcframework``.
+                -   In case you need multitarget library usage, filename should be ``MyPlugin.a`` and ``a`` files should be named as ``MyPlugin.release.a`` and ``MyPlugin.debug.a``.
```

Comment:
```suggestion
                -   In case you need multitarget library usage, the filename should be ``MyPlugin.a`` and ``.a`` files should be named as ``MyPlugin.release.a`` and ``MyPlugin.debug.a``.
```